apps/ledger-live-mobile/src/dynamicContent/ContentCardsLocation.tsx

```diff
@@ -37,7 +43,13 @@ const ContentCardsLocationComponent = ({ locationId, hasStickyCta, ...containerP
         renderItem={Category}
         keyExtractor={(item: CategoriesWithCards) => item.category.id}
         ItemSeparatorComponent={() => <Flex height={32} />}
-        ListFooterComponent={hasStickyCta ? <Flex height={116} /> : null}
+        ListFooterComponent={
+          hasStickyCta ? (
+            <Flex height={116} />
+          ) : bottomSpacing ? (
+            <Flex height={bottomSpacing} />
+          ) : null
+        }
```

Comment:
we should had a margin anyway if no sticky no?

Reply:
That's what it does, if there is no sticky cta and we passed a `bottomSpacing` parameter, we'll have a spacing in the bottom

Reply:
But we can choose to not send a `bottomSpacing` parameter to not have any spacing in the bottom of the `ContentCardsLocation` component